Docs/Generali/norme_di_progetto/05_std_qualita.tex

```diff
@@ -0,0 +1,279 @@
+\section{Standard per la qualità}
+
+\subsection{ISO/IEC 12207:1995}
+Per definire, misurare e valutare i processi da attuare nello svolgimento
+del progetto, il gruppo adotta lo standard ISO/IEC 12207:1995, che
+definisce un modello per i processi di ciclo di vita del software. 
+Tale modello suddivide i processi in primari, di supporto e organizzativi.
+Ogni processo consiste di più attività, e ogni attività è a sua volta
+composta da più task.
```

Comment:
Direi: Ogni processo si suddivide in sottoprocessi, ciascuno dei quali è a sua volta composto da più attività.

Reply:
Each life cycle process is divided into a set of activities; each activity is further divided into a set of tasks.
Questo è preso dallo standard https://www.math.unipd.it/~tullio/IS-1/2009/Approfondimenti/ISO_12207-1995.pdf

Reply:
ok